vignettes/future-4-non-exportable-objects.md.rsp

```diff
@@ -606,11 +606,11 @@ and
 ```r
 file <- system.file("ex/elev.tif", package = "terra")
 r <- rast(file)
-.r <- wrap(v)
-dr %<-% { r <- vect(.r); dim(r) }
+.r <- wrap(r)
+dr %<-% { r <- rast(.r); dim(r) }
```

Comment:
This should be `unwrap()`, correct?

Reply:
Good catch. Yes, I think using `unwrap()` better reflect the relationship between `wrap()` and `unwrap()`.  But I see that I also write `wrap()` and `vect()` above; maybe I got that from the example in `?wrap`.

I'll go ahead an merge and make these tweaks myself.